securing_apps/topics/client-registration.adoc

```diff
@@ -1,5 +1,5 @@
 [[_client_registration]]
-== Client Registration
+== Using the Client Registration service
```

Comment:
Optional: `Service` is ok and also used in other places. But sounds more natural to me `Endpoint` because it is more aligned with the specs.